[2023-08-02 09:35:44,937] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: S3_to_BigQuery.load_weather_data manual__2023-08-02T09:35:42.964909+00:00 [queued]>
[2023-08-02 09:35:44,951] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: S3_to_BigQuery.load_weather_data manual__2023-08-02T09:35:42.964909+00:00 [queued]>
[2023-08-02 09:35:44,951] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2023-08-02 09:35:44,951] {taskinstance.py:1357} INFO - Starting attempt 1 of 2
[2023-08-02 09:35:44,951] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2023-08-02 09:35:44,975] {taskinstance.py:1377} INFO - Executing <Task(PythonOperator): load_weather_data> on 2023-08-02 09:35:42.964909+00:00
[2023-08-02 09:35:44,981] {standard_task_runner.py:52} INFO - Started process 10502 to run task
[2023-08-02 09:35:44,984] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'S3_to_BigQuery', 'load_weather_data', 'manual__2023-08-02T09:35:42.964909+00:00', '--job-id', '323', '--raw', '--subdir', 'DAGS_FOLDER/dag.py', '--cfg-path', '/tmp/tmpt012ninj', '--error-file', '/tmp/tmpwo8m1sxo']
[2023-08-02 09:35:44,984] {standard_task_runner.py:80} INFO - Job 323: Subtask load_weather_data
[2023-08-02 09:35:45,043] {task_command.py:370} INFO - Running <TaskInstance: S3_to_BigQuery.load_weather_data manual__2023-08-02T09:35:42.964909+00:00 [running]> on host 656341a29da6
[2023-08-02 09:35:45,139] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=S3_to_BigQuery
AIRFLOW_CTX_TASK_ID=load_weather_data
AIRFLOW_CTX_EXECUTION_DATE=2023-08-02T09:35:42.964909+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-08-02T09:35:42.964909+00:00
[2023-08-02 09:35:45,625] {logging_mixin.py:115} INFO - 
DataFrame push into xcom.
[2023-08-02 09:35:45,628] {logging_mixin.py:115} INFO - 
DATAFRAME SHAPE : (13, 11)
[2023-08-02 09:35:45,628] {logging_mixin.py:115} INFO - 
✅ DataFrame created ! ✅
[2023-08-02 09:35:46,553] {logging_mixin.py:115} INFO - 
✅ Push to xcom ! ✅
[2023-08-02 09:35:46,555] {python.py:173} INFO - Done. Returned value was: None
[2023-08-02 09:35:46,634] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=S3_to_BigQuery, task_id=load_weather_data, execution_date=20230802T093542, start_date=20230802T093544, end_date=20230802T093546
[2023-08-02 09:35:46,799] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-08-02 09:35:46,869] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
